src/pages/UserPreference.tsx

```diff
@@ -0,0 +1,77 @@
+import { ReactElement } from "react";
+import "tw-elements";
+
+export type Member = {
+  id: String;
+  firstName: String;
+  lastName: String;
+  email: String;
+  activeMember: Boolean;
+  votingRights: Boolean;
+  receiveNotPresentEmail: Boolean;
+  includeInQuorum: Boolean;
+  signInBlocked: Boolean;
+};
+
+export type UserPreferenceProp = {
+  member: Member;
+};
+
+const UserPreference = ({ member }: UserPreferenceProp): ReactElement => {
+  return (
+    <div className="flex flex-col flex-1 p-4 font-sans md:p-10 gap-y-8">
+      <div className="flex flex-col font-sans font-bold">
+        <span className="text-gray-600 text-xl"> Hello,</span>
+        <span className="section-heading m-0"> NORTHEASTERN SGA</span>
+      </div>
+
+      <div className="flex flex-col px-7 font-sans gap-8">
+        <span className="font-bold text-xl">CONTACT INFO</span>
+        <div>
+          <span className="text-gray-600">Name</span> <br />
+          {member.firstName + " " + member.lastName}
+        </div>
+        <div className="flex flex-col">
+          <span className="text-gray-600">Email</span>
+          <span>{member.email}</span>
+        </div>
+
+        <hr className="border-black home-mx -ml-7 -mr-7" />
```

Comment:
you don't need to use `home-mx` here, then u shouldnt need to use the `-ml` and `-mr` either !

Reply:
just for future reference what was the purpose of home-mx I saw it in the design a couple times for a couple of <hr />, and I figured I should've used them for mine

Reply:
Oh it's just a class I created for creating margins on the borders of the homepage, since I was having to use a specific margin on the homepage i just turned it into a class - it probably doesn't need to be turned into a class though lol